### PR TITLE
created transformation script

### PR DESCRIPTION
**Pull request by twarep**
## Checklist
- [x] Did I add this pull request into the appropriate Notion ticket? If not, create a ticket for this PR.

## What I Did
Created a file called transformation.py. It contains three methods:

1. **transform_overview_data(df)** - takes the dataframe representing the overview and uses the helper methods 2 and 3 to clean the data
2. **drop_cell_temps(df)** - drops all cell temperatures from "bat 1 cell 1" to "bat 2 cell 16"
3. **drop_remaining_columns(df)** - drops all remaining columns that should be removed. See code for a list

Added the transform_overview_data(df) function to scraper.py, using it after each csv is converted into a pandas df.

## How I Did it?
Used pandas package. Separated transformation script into its own file for readability and debugging purposes.

## How to Test?
Uncomment the print(df.head()) method to check that for each download

### Relevant Links
None.

### CODE REVIEW

**Comment by NayeemaNonta** [comment]
You have to remove these columns: 'bat 1 min cell volt', 'bat 2 min cell volt', 'bat 1 max cell volt', 'bat 2 max cell volt'. 

And you're missing: remaining_flight_time, OAT 


This is my output for all the columns after running the file, I cross-referenced with the [Data Breakdown Sheet](https://onedrive.live.com/edit.aspx?resid=49087F5D0DCC286B!36669&ithint=file%2cxlsx&authkey=!ACBK4-EGqVZdokE)

Screen Shot 2023-07-17 at 9 40 20 PM

File: pipeline/transformation.py [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+      df = df.drop(current_battery_temp, axis=1)
+  return df
+
+# drops the remaining columns (manually)
+def drop_remaining_columns(df):
+  df = df.drop("time(ms)", axis=1)
+  df = df.drop("inverter operating time", axis=1)
+  df = df.drop("stall_pressure_diff_raw", axis=1)
+  df = df.drop("stall_calibrated_value", axis=1)
+  df = df.drop("remaining flight time", axis=1)
```
Comment (NayeemaNonta) [inline pipeline/transformation.py]: keep remaining_flight_time

File: pipeline/transformation.py [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+  df = df.drop("ACC_LONG", axis=1)
+  df = df.drop("ACC_LAT", axis=1)
+  df = df.drop("ACC_NORM", axis=1)
+  df = df.drop("OAT", axis=1)
```
Comment (NayeemaNonta) [inline pipeline/transformation.py]: keep OAT

**Comment by twarep** [comment]
'bat 1 min cell volt', 'bat 2 min cell volt', 'bat 1 max cell volt', 'bat 2 max cell volt' are in
[columns final maybe](https://onedrive.live.com/edit.aspx?resid=49087F5D0DCC286B!36669&ithint=file%2cxlsx&authkey=!ACBK4-EGqVZdokE)

remaining_flight_time and OAT are in [columns final remove](https://onedrive.live.com/edit.aspx?resid=49087F5D0DCC286B!36669&ithint=file%2cxlsx&authkey=!ACBK4-EGqVZdokE)
 
Please confirm that we don't need/need these columns and I'll make the change.

**Comment by NayeemaNonta** [comment]
Based on today's standup I think the columns final keep sheet is updated, but not the other two. I will let you know once @joyang15 or @vik01 confirms the correct lists.

**Review by NayeemaNonta — APPROVED** [review]
Looks good, great work :)

**Pull request merged**